automated/linux/armnn/armnn-unit-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:a:b:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    a) LINK_SNAPSHOT_32BIT="${OPTARG}" ;;
+    b) LINK_SNAPSHOT_64BIT="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
+
+pkgs="ntp wget gcc g++ python3 python3-pip"
+dhclient
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+#(GET WHICH PLATFORM)
+detect_abi
+
+if [ "$abi" == "armeabi" ]
+then
+    mkdir /usr/local/bin && cd /usr/local/bin && \
+    wget "${LINK_SNAPSHOT_32BIT}" && tar xf armnn-32bit.tar.xz && \
+    export BASEDIR="/home/buildslave/workspace/armnn32-ci-build"
+else
+    wget "${LINK_SNAPSHOT_64BIT}" && tar xf armnn.tar.xz && \
+    export BASEDIR="home/buildslave/workspace/armnn-ci-build"
+
+cd $BASEDIR/build
+ln -s libprotobuf.so.15.0.0 ./libprotobuf.so.15
+export LD_LIBRARY_PATH=`pwd`
+chmod a+x UnitTests
```

Comment:
these four lines shouldn't be done if we set 'SKIP_INSTALL=true', and don't pass in any tarball right (which implies that the armnn is already in the rootfs...